src/main/java/com/robin/hoteless/model/Reservation.java

```diff
@@ -0,0 +1,119 @@
+package com.robin.hoteless.model;
+
+import java.time.LocalDate;
+import java.util.Date;
+
+public class Reservation {
+    private int id;
+    private Date reservationCreationDate;
```

Comment:
Since the class is Reservation, you can probably name this just ``creationDate`` and it would be perfect.